components/SalesHistoryTable/index.tsx

```diff
@@ -71,30 +73,44 @@ const getAvatars = async (
 const getMySalesHistory = async ({
   id,
```

Comment:
Let's rename this `id` to `templateId` and have `SalesHistoryTable` take in a `templateId` prop so it's easier to know what type of ID is being referenced.